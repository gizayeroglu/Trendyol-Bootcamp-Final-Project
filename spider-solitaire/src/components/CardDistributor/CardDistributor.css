.card-distrubitor-container {
  display: grid;
  align-items: flex-start;
  grid-template-columns: repeat(auto-fill, 20%);
  padding-right: 10px;
}
.distubitor-card {
  height: 130px; 
  width: 95px;
  border-radius: 10px;
  border: none;
  cursor: pointer;
  background: url("../../assets/spider-solitaire-card-back.webp");
  background-size: cover;
  background-repeat: no-repeat;
  background-position: center center;
}

@media only screen and (min-width:800px) and (max-width: 1200px) {
  .distubitor-card {
    height: 90px;
    width: 65px;
  }
}

@media only screen and (min-width:500px) and (max-width:800px) {
  .distubitor-card {
    height: 65px;
    width: 45px;
    border-radius: 5px;
  }
}

@media only screen and (max-width:500px) {
  .distubitor-card {
    height: 50px;
    width: 35px;
    border-radius: 5px;
  }
}