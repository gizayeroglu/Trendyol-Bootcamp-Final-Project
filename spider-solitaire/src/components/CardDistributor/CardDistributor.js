import './CardDistributor.css';

import React from 'react';

function CardDistributor ({ getNewCards }) {
  return (
    <>
      <div className='card-distrubitor-container'>
        <div className='distubitor-card' onClick={getNewCards}></div>
        <div className='distubitor-card' onClick={getNewCards}></div>
        <div className='distubitor-card' onClick={getNewCards}></div>
        <div className='distubitor-card' onClick={getNewCards}></div>
        <div className='distubitor-card' onClick={getNewCards}></div>
      </div>
    </>
  );
}

export default CardDistributor ;
