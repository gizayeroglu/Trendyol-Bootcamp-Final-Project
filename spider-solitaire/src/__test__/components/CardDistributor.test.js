import React from "react";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import { configure, shallow, mount } from "enzyme";

import CardDistributor from "../../components/CardDistributor/CardDistributor";

configure({ adapter: new Adapter() });

describe("Card Distrubitor component tests", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<CardDistributor  />);
  });

  it("should render the card holder component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
