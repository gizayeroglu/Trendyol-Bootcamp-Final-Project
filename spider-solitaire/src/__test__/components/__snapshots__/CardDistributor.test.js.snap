// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Distrubitor component tests should render 1`] = `ShallowWrapper {}`;
